apps/editors/templates/editors/queue.html

```diff
@@ -27,7 +27,7 @@
         <button type="submit">{{ _('Search') }}</button>
         <a class="toggle-queue-search" href="#">{{ _('Advanced Search') }}</a>
         {% if search_form.data and search_form.data['searching'] %}
-          (<a class="clear-queue-search" href="{{ url('editors.queue_%s' % tab) }}">{{ _('clear search') }}</a>)
+        (<a class="clear-queue-search" href="{{ url('editors.unlisted_queue_{0}'.format(tab)) if unlisted else url('editors.queue_{0}'.format(tab)) }}">{{ _('clear search') }}</a>)
```

Comment:
This is a bit hard to parse. I think `{{ url('editors.{1}queue_{0}'.format(tab, ('unlisted_' if unlisted else ''))) }}` is slightly more readable, but not a big deal either way.